test/HostActivationTests/project.json

```diff
@@ -3,7 +3,7 @@
   "dependencies": {
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0"
+      "version": "1.1.0"
```

Comment:
This was already existing, but we are depending on Microsoft.NETCore.App version `1.1.0`, but using TFM "netcoreapp2.0".  This seems incorrect.